components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -3571,16 +3584,10 @@ subroutine rain_sedimentation(kts,kte,ktop,kbot,kdir,   &
 
          enddo kloop_sedi_r1
 
-         if (k_qxbot.eq.kbot) then
```

Comment:
@pressel , @AaronDonahue , while I was looking at the code, it seemed to me that this block of code is not needed. k_qxbot is modified in the same way as k_temp within generalized sedimentation, so we can just use it instead of k_temp.

Reply:
@jgfouca are you sure. In generalized_sedimentation, k_qxbot is updated at the end with `if (k_qxbot.ne.kbot) k_qxbot = k_qxbot - kdir`, I believe making it potentially different than k_temp.

Reply:
@ambrad , isn't that the same thing that's happening here for k_temp, just with the logic inverted?

```
if (k_qxbot.eq.kbot) then
   k_temp = k_qxbot
else
   k_temp = k_qxbot-kdir
endif
```
vs.
```
if (k_qxbot.ne.kbot) k_qxbot = k_qxbot - kdir
...
k_temp = k_qxbot
```

Reply:
Ok, great.